Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditableRow.cs

```diff
@@ -466,13 +466,15 @@ public IBusinessObjectBoundEditableWebControl[] GetEditControlsAsArray ()
           {
             ((HtmlControl)validationErrorMarker).Attributes["tabIndex"] = "0";
             ((HtmlControl)validationErrorMarker).Attributes["title"] = null;
-            ((HtmlControl)validationErrorMarker).Attributes[HtmlTextWriterAttribute2.AriaLabel] = toolTipBuilder.ToString();
+            if (!_editModeHost.DisableEditModeValidationMessages)
```

Comment:
```suggestion
            if (_editModeHost.DisableEditModeValidationMessages)
```

Please verify, I believe you got an negation error here.

Reply:
Don't think so. We only want the tooltip if the validation messages have not been disabled -> `!DisableEditModeValidationMessages`

Reply:
With the suggestion, there is no tooltip in 
![image](https://github.com/re-motion/Framework/assets/6972548/06e9a0e5-7cf3-4ad9-9006-b047ab28a25c)

With the original code, the tooltip is generated. Check also line ~506 where we render the validation message based on `DisableEditModeValidationMessages` being `false`. The updated code needs to compensate/use inverted semantics.